plugins/GSdx/GSRendererHW.cpp

```diff
@@ -201,22 +201,23 @@ void GSRendererHW::InvalidateLocalMem(const GIFRegBITBLTBUF& BITBLTBUF, const GS
 	m_tc->InvalidateLocalMem(m_mem.GetOffset(BITBLTBUF.SBP, BITBLTBUF.SBW, BITBLTBUF.SPSM), r);
 }
 
-int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
+uint16 GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 {
 	float t = (1.0f - alpha) * t0 + alpha * t1;
-	return (int)t & ~0xF; // cheap rounding
+	return (uint16)t & ~0xF; // cheap rounding
 }
```

Comment:
In any case the cast would not disappear. It would just move one line upwards. The only difference would be the reduced memory load of a single(32bit) vs an uint16.